tests/testsuite.src/run_extensions.at

```diff
@@ -1629,47 +1629,47 @@ AT_DATA([prog.cob], [
        PROCEDURE        DIVISION.
 *     *    MOVE NON-INTEGER TO ALPHA-NUMERIC --> ignore Decimal Point!
        S-01.
-          MOVE SPACES TO ALPHA-FIELD.
-          MOVE INTEGER TO ALPHA-FIELD.
-          DISPLAY  ALPHA-FIELD NO ADVANCING
-          END-DISPLAY.
+           MOVE SPACES TO ALPHA-FIELD.
```

Comment:
please ensure one of those "code in column 10" is in the syntax tests below, compiled with -std=default/cobol2002/mf (should compile without warnings) and cobol85/gcos (should likely fail compilation) - for the later MF (with dialect ANS85) errors with

```
Word starts or is continued in wrong area of source line
```

Something similar would be nice for area-a enforcement